src/Kernel-CodeModel/Class.class.st

```diff
@@ -191,11 +191,10 @@ Class >> allSharedPools [
 
 { #category : 'class variables' }
 Class >> anyUserOfClassVarNamed: aSymbol [
```

Comment:
This seems really specific indeed I think we can deprecate it

Reply:
is was used to find if a class variable was used in code. For that of course we have a better API on the Variable